src/screens/MyPromotions.tsx

```diff
@@ -62,94 +63,14 @@ const dropdowns: Dropdown[] = [
 ];
 
 const user: User = {
-  id: 'u1',
+  id: '8f8fc016-5bb2-4906-ad88-68932c438665',
```

Comment:
Ah so this will only work if everyone has a user with `id: '8f8fc016-5bb2-4906-ad88-68932c438665'`. I guess this is temporary until we have user auth implemented

Reply:
Yeah, I couldn't think of another solution for now. I just took this id from the init_data.sql file, so hopefully everyone should have it if they have their db up-to-date? Not sure how often this file changes as well